### DISCUSSION TITLE
OBS crashes on closing

### DISCUSSION THREAD

**Discussion by AlfaTelevision** [obsproject/obs-studio#6034] [Development Questions] [answered]
Hi I have a problem with v 27.2.1 crashes and freezes the whole computer. How can you do it?

**Comment by feetstv** [comment]
This is not the ideal place for support. Visit the [OBS Discord](https://obsproject.com/discord) or [support forums](https://obsproject.com/forum) to get support for OBS.

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
